src/Files.App/Views/Properties/CompatibilityPage.xaml

```diff
@@ -7,113 +7,186 @@
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:helpers="using:Files.App.Helpers"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	xmlns:settingsuc="using:Files.App.UserControls.Settings"
 	xmlns:vm="using:Files.App.ViewModels.Properties"
-	Loaded="Properties_Loaded"
 	Tag="Compatibility"
 	mc:Ignorable="d">
 
 	<vm:BasePropertiesPage.Resources>
 		<ResourceDictionary>
 			<ResourceDictionary.MergedDictionaries>
 				<ResourceDictionary Source="ms-appx:///ResourceDictionaries/PropertiesStyles.xaml" />
-				<ResourceDictionary Source="/ResourceDictionaries/RightAlignedToggleSwitchStyle.xaml" />
 			</ResourceDictionary.MergedDictionaries>
 
+			<converters:EnumToBoolConverter x:Key="EnumToBoolConverter" />
 			<converters:NullToTrueConverter x:Key="NullToFalseConverter" Inverse="True" />
 
 		</ResourceDictionary>
 	</vm:BasePropertiesPage.Resources>
 
-	<ScrollViewer x:Name="RootScrollViewer">
+	<ScrollViewer>
 		<StackPanel Padding="12" Spacing="4">
 
-			<!--  Run Troubleshooter  -->
-			<settingsuc:SettingsBlockControl
-				Title="{helpers:ResourceString Name=RunCompatibilityTroubleShooter}"
-				ButtonCommand="{x:Bind CompatibilityProperties.RunTroubleshooterCommand}"
-				IsClickable="True">
-				<FontIcon
-					FontSize="14"
-					Foreground="{ThemeResource TextFillColorPrimaryBrush}"
-					Glyph="&#xE8A7;" />
-			</settingsuc:SettingsBlockControl>
+			<!--  Launch Compatibility Troubleshooter  -->
+			<StackPanel
+				Padding="16"
+				Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+				BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+				BorderThickness="1"
+				CornerRadius="{StaticResource ControlCornerRadius}"
+				Spacing="8">
+
+				<TextBlock x:Name="RunCompatibilityTroubleshooterDescriptionTextBlock" Text="{helpers:ResourceString Name=CompatibilityRunTroubleshooterDescription}" />
+
+				<Button
+					x:Name="RunCompatibilityTroubleshooterButton"
+					Command="{x:Bind CompatibilityViewModel.RunTroubleshooterCommand}"
+					Content="{helpers:ResourceString Name=CompatibilityRunTroubleshooter}" />
+
+			</StackPanel>
+
+			<!--  Section (Compatibility Mode)  -->
+			<TextBlock
+				x:Name="CompatibilityModeSectionHeaderTextBlock"
+				Margin="4,12,4,0"
+				Style="{StaticResource BodyStrongTextBlockStyle}"
+				Text="{helpers:ResourceString Name=CompatibilityMode}" />
 
 			<!--  Compatibility Mode  -->
-			<settingsuc:SettingsBlockControl Title="{helpers:ResourceString Name=CompatibilityMode}" HorizontalAlignment="Stretch">
+			<StackPanel
+				Padding="16"
+				Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+				BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+				BorderThickness="1"
+				CornerRadius="{StaticResource ControlCornerRadius}"
+				Spacing="8">
+
+				<TextBlock x:Name="CompatibilityModeDescriptionTextBlock" Text="{helpers:ResourceString Name=CompatibilityModeDescription}" />
```

Comment:
These should be the **Header** property of the **ComboBox**.

As I mentioned some time ago, somewhere else. I wonder why you guys don't use the **x:Uid** property. It makes translation easy. You can set it like this:
```xaml
 <ComboBox x:Uid="CompatibilityModeComboBox" ...>
```
and the set this in the resources file:
```xml
<data name="CompatibilityModeComboBox.Header">
  <value>Compatibility mode</value>
</data>
```
Bonus: you can set all kinds of properties via one **Uid**, not just text.
https://learn.microsoft.com/windows/uwp/app-resources/localize-strings-ui-manifest

Reply:
Let me try.

Reply:
Let me try. We don't use Uid because it prevents us from re-suing it again somewhere else. I added "Compatibility" as a prefix so someone who work on this field can easily find resources in resx viewer tho.

Reply:
Done